pkgs/tools/security/fido2luks/default.nix

```diff
@@ -9,13 +9,13 @@
 
 rustPlatform.buildRustPackage rec {
   pname = "fido2luks";
-  version = "0.2.19";
+  version = "0.2.20";
 
   src = fetchFromGitHub {
     owner = "shimunn";
     repo = pname;
-    rev = version;
-    sha256 = "sha256-o21KdsAE9KznobdMMKfVmVnENsLW3cMZjssnrsoN+KY=";
+    rev = "210da1ce0fdd8a7259329b9cff636b4ab9929936";
```

Comment:
https://github.com/shimunn/fido2luks/tree/0.2.20

Reply:
Please keep `rev = version;` - the referenced commit has been tagged as `0.2.20` so it should work correctly.

Reply:
Hmm yeah, not sure what I was thinking there. I've amended the first commit to keep `rev = version;`.